packages/stryker/src/ConfigValidator.ts

```diff
@@ -48,6 +57,27 @@ export default class ConfigValidator {
     }
   }
 
+  private validateLogLevel() {
+    const logLevel = this.strykerConfig.logLevel;
+    const VALID_LOG_LEVEL_VALUES = ['fatal', 'error', 'warn', 'info', 'debug', 'trace', 'all', 'off'];
+    if (VALID_LOG_LEVEL_VALUES.indexOf(logLevel) < 0) {
+      this.invalidate('\`logLevel\` is invalid, expected one of \`fatal\`, \`error\`, \`warn\`, \`info\`, \`debug\`, \`trace\`, \`all\` and \`off\`');
```

Comment:
You could use `VALID_LOG_LEVEL_VALUES .join(',')` here to reduce duplication.

Reply:
This is actually done in `validateCoverageAnalysis`. I'm a fan on using the join here as well.